data/map planets.txt

```diff
@@ -4811,6 +4857,9 @@ planet "Varu Tek'kai"
 	outfitter "Wanderer Basics"
 	"required reputation" 15
 	bribe 0
+	tribute 200
+		threshold 1300
```

Comment:
```suggestion
		threshold 1500
```
Again, 1500 seems to be the minimum so far, though this one seems more reasonable than the 350 on Var'roi.